src/server/pages.js

```diff
@@ -2,6 +2,8 @@ import nextRoutes from 'next-routes';
 
 const pages = nextRoutes();
 
+const createOrderPage = process.env.USE_OLD_CREATE_ORDER === 'true' ? 'createOrder' : 'createOrderNewFlow';
```

Comment:
Should I use `getEnvVar` ?

Reply:
Possibly. Then you need to add the env variable to the list in `_document.js`.

Just in case, feel free to propose a better solution as I'm not happy with this one.

Reply:
I didn't, just added `USE_NEW_CREATE_ORDER: process.env.USE_NEW_CREATE_ORDER || false` to `_document.js`